github/gists.go

```diff
@@ -52,6 +52,29 @@ func (g GistFile) String() string {
 	return Stringify(g)
 }
 
+// GistCommit represents a commit on a gist.
+type GistCommit struct {
+	Version      *string        `json:"version,omitempty"`
+	User         *User          `json:"user,omitempty"`
+	ChangeStatus map[string]int `json:"change_status,omitempty"`
+	CommitedAt   *Timestamp     `json:"commited_at,omitempty"`
+}
+
+func (gc GistCommit) String() string {
+	return Stringify(gc)
+}
+
+type GistFork struct {
```

Comment:
Since this is an exported symbol, it needs to be documented.

Source: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

> All top-level, exported names should have doc comments

It's also something go lint would catch:

```Go
github/gists.go:67: exported type GistFork should have comment or be unexported
```